airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -221,19 +216,43 @@ def _upload_to_gcs(self, files_to_upload):
         for object, tmp_file_handle in files_to_upload.items():
             hook.upload(self.bucket, object, tmp_file_handle.name, 'application/json')
 
-    @classmethod
-    def convert_types(cls, value):
+    def _convert_types(self, schema, row):
         """
         Takes a value from MySQLdb, and converts it to a value that's safe for
         JSON/Google cloud storage/BigQuery. Dates are converted to UTC seconds.
-        Decimals are converted to floats.
+        Decimals are converted to floats. Binary type fields are encoded with base64.
+        As imported BYTES data must be base64-encoded, according to Bigquery SQL
+        date type documentation: https://cloud.google.com/bigquery/data-types
+        """
+        col_type_dict = self._get_col_type_dict()
```

Comment:
the mapping should be done in the constructor, reconstructing it on each row is inefficient since it's static

Reply:
good point